components/lists/Signaling.tsx

```diff
@@ -19,76 +20,98 @@ interface SignalingProps {
-      <View style={styles.container}>
-        <View
-          style={{
-            flexDirection: "row",
-            height: 50,
-            backgroundColor: theme.colors.black,
-            gap: 12,
-          }}>
-          <Badge variant="primary" name="100" />
-          <Badge variant="primary" name="3/4" style={{ opacity: 0.3 }} />
-        </View>
-        <View style={styles.header}>
-          <Badge name={displayUsers.length.toString()} />
-          <CustomText size="sm" fontWeight="bold">
-            Friends are signaling
-          </CustomText>
-        </View>
-        <FlatList
-          contentContainerStyle={styles.listContent}
-          style={styles.flatList}
-          data={displayUsers as User[]}
-          keyExtractor={(item) => item.id}
-          renderItem={({ item }) => (
-            <View style={styles.userCard}>
-              <UserAvatar imageUrl={item.imageUrl || 0} />
-              <View>
-                <UserInfo
-                  firstName={item.firstName}
-                  lastName={item.lastName}
-                  username={item.username}
-                />
-              </View>
-            </View>
-          )}
+      <View style={styles.header}>
+        <CustomText size="xl" fontWeight="bold" style={styles.headerText}>
+          Friends
+        </CustomText>
+        {/* TODO: this should redirect to the search screen */}
+        <CustomButton
+          variant="primary"
+          textSize="sm"
+          title="FIND"
+          textStyles={{ fontWeight: 600 }}
         />
       </View>
+      {!availableUsers.length && (
+        <CustomText style={styles.noUsers}>
+          None of your friends on Wavv are available today
+        </CustomText>
+      )}
+      <BottomSheetSectionList
+        sections={[
+          {
+            title: "available users",
+            data: availableUsers,
+            renderItem: ({ item }) => (
+              <View style={styles.userCard}>
+                <UserAvatar imageUrl={item.imageUrl || 0} />
+                <View>
+                  <UserAvailability
+                    firstName={item.firstName}
+                    lastName={item.lastName}
+                    time={item.time}
+                    activity={item.activity}
+                  />
+                </View>
+              </View>
+            ),
+          },
+          {
+            title: "Other users",
+            data: otherusers,
+            renderItem: ({ item }) => (
+              <View style={[styles.userCard, styles.availableUserCard]}>
+                <UserAvatar imageUrl={item.imageUrl || 0} />
+                <View>
+                  <UserInfo
+                    firstName={item.firstName}
+                    lastName={item.lastName}
+                    username={item.username}
+                  />
+                </View>
+              </View>
+            ),
+          },
+        ]}
+        keyExtractor={(item) => item.id}
+      />
```

Comment:
can we have the two renderItems defined outside, think it would improve the readability.
```suggestion

....
const renderAvailableUser=()=>(
 <View style={styles.userCard}>
                <UserAvatar imageUrl={item.imageUrl || 0} />
                <View>
                  <UserAvailability
                    firstName={item.firstName}
                    lastName={item.lastName}
                    time={item.time}
                    activity={item.activity}
                  />
                </View>
              </View>)
const renderOtherUser=()=>(
  <View style={[styles.userCard, styles.availableUserCard]}>
                <UserAvatar imageUrl={item.imageUrl || 0} />
                <View>
                  <UserInfo
                    firstName={item.firstName}
                    lastName={item.lastName}
                    username={item.username}
                  />
                </View>
              </View>)
  
.... 
return(
....
      <BottomSheetSectionList
        sections={[ { title: "available users", data: availableUsers,renderAvailableUser  },
         { title: "Other users", data: otherUsers, renderItem: renderOtherUser } ]}
        keyExtractor={(item) => item.id}
      />
```

-  Also why not call them onlineUsers and offlineUsers, this would make it clearer who these users are. Or if we stick to availableUsers and otherUsers, we use the same names in the whole component